pkg/middleware/stored_session.go

```diff
@@ -40,14 +42,6 @@ type StoredSessionLoaderOptions struct {
 
 	// How often should sessions be refreshed
 	RefreshPeriod time.Duration
-
-	// Provider based session refreshing
-	RefreshSession func(context.Context, *sessionsapi.SessionState) (bool, error)
-
-	// Provider based session validation.
-	// If the sesssion is older than `RefreshPeriod` but the provider doesn't
-	// refresh it, we must re-validate using this validation.
-	ValidateSession func(context.Context, *sessionsapi.SessionState) bool
```

Comment:
This seems like a good idea 🤔

Reply:
would you elaborate please.
I removed this since now we're loading tenant/provider from request context and calling the provider's `RefreshSession` and `ValidateSession` methods.